src/apps/tools/OswAppStepStats.cpp

```diff
@@ -0,0 +1,97 @@
+
+#include "./apps/tools/OswAppStepStats.h"
+#include "./apps/watchfaces/OswAppWatchfaceFitness.h"
+
+#include <gfx_util.h>
+#include <osw_app.h>
+#include <osw_hal.h>
+#include <osw_pins.h>
+
+void OswAppStepStats::drawChart() {
+#if OSW_FEATURE_STATS_STEPS
```

Comment:
Same, temporary method for eliminating compilation errors.

Reply:
↑